apps/outlook-rooms-addin/src/app/rooms/find-space/find-space.component.ts

```diff
@@ -126,27 +128,39 @@ export class FindSpaceComponent extends BaseClass implements OnInit {
         await this._org.initialised.pipe(first((_) => !!_)).toPromise();
         await this._spaces.initialised.pipe(first((_) => !!_)).toPromise();
         await this._state.available_spaces.pipe(take(1)).toPromise();
+        this.spaces$ = this._state.available_spaces;
+
+        this.spaces$.subscribe((i) => console.log(i, 'avail spaces'));
 
         this.setBuilding(this._org.building);
         this.book_space = {};
 
         await this._mapService.locateSpaces(this.spaces$);
 
         this.locatable_spaces$ = this._mapService.locatable_spaces$;
+
         this.maps_list$ = this._mapService.maps_list$;
 
+        //testing multiple levels//
+        // this.maps_list$ = of([
+        //     {
+        //         map_id: 'assets/maps/level_1.svg',
+        //         level: 'Level 1',
+        //     },
+        //     {
+        //         map_id: 'assets/maps/level_2.svg',
+        //         level: 'Level 2',
+        //     },
+        // ]);
+        //end of testing multiple levels//
+
+        this.maps_list$.subscribe((maps) => (this.selected_level = maps));
```

Comment:
If the intend is to have an after effect of the `maps_list$` observable, better to use the `tap` operation than creating another subscription (which is unmanaged when the component is destroyed btw = potential memory leak)
Example:
```
Line 136: this.maps_list$ = this._mapService.maps_list$.pipe( tap((maps) => this.selected_level = maps) );
```